### DISCUSSION TITLE
Pen hover is Affected by Sensitivity.

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27684] [Q&A]
### Type

Game behaviour

### Bug description

Pen hover overcompensates or undercompensates from the center based on Cursor sensitivity on 2 in one laptops. actual inputs are not affected, but this may confuse some players.

### Screenshots or videos

[Hardware Demonstration.webm](https://github.com/ppy/osu/assets/144076671/6ccd7a09-00af-4cd0-90f4-7eb9ef382195)
[Software Camera.webm](https://github.com/ppy/osu/assets/144076671/ce08a310-a651-47fe-95ec-e4af75301a51)

### Version

2024.312.1-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14663210/compressed-logs.zip)

**Comment by Hissymaster** [comment]
the software cam is laggy as heck because my laptop is underpowered. but the demonstration is still there at least.

**Comment by bdach** [comment]
I'm not sure we can do anything about this? This may as well be a device driver being misdesigned (it's likely emitting mouse events on pen hover rather than absolute pointing device events).

Definitely would require access to the hardware to even begin diagnosing this. May be an upstream issue too.

**Reply by Hissymaster** [reply]
Honestly, that would probably be right. Sorry for bothering you.

**Reply by Hissymaster** [reply]
I didn't really realise it was on a device by device basis. Again, so sorry.
